app/forms/sufia/forms/admin_set_form.rb

```diff
@@ -3,6 +3,7 @@ module Forms
     class AdminSetForm < CurationConcerns::Forms::CollectionEditForm
       self.model_class = AdminSet
       self.terms = [:title, :description, :thumbnail_id]
+      attr_reader :workflow_name
```

Comment:
Without this, I was getting an error thrown on the form. Delegation may fix that, I'll work on it.